Mifare Classic Tool/gradlew

```diff
@@ -1,79 +1,129 @@
-#!/usr/bin/env bash
```

Comment:
Why replace it with just `/bin/sh`? The old format made more sense to me.
Did you do the big grade update yourself or is there an automatic tool for it?

Reply:
Generate gradle wrapper by running `./gradlew wrapper`.